.github/CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 # Contributions
 
-ClassicPress is a volunteer-driven open source project that thrives when we have multiple contributors working towards a common goal.
+ClassicPress is a volunteer-driven open source project that thrives when we have multiple contributors working towards a common goal. 
+
+There may be some delays in reviewing and merging pull requests. This is because we take a lot of care with our platform and ClassicPress is maintained by volunteers in our free time.
```

Comment:
How about just removing this sentence instead? The delay for reviewing PRs should be shortened when we have more people looking at them

Reply:
50/50 for me. One of the most unexpected things in FOSS is people assuming everything is snap. This is a gentle reminder. But we can agree to disagree.

Reply:
Maybe we can make the reminder more gentle then:

```suggestion
We invite all contributors to not only submit pull requests that are in line with the goals and direction of the project, but also to discuss and review other contributors' issues and pull requests. This will help us make the most of our limited time and review all contributions quickly.
```